server/consumer.go

```diff
@@ -1991,8 +1995,12 @@ func (o *consumer) processAck(subject, reply string, hdr int, rmsg []byte) {
 		o.processNak(sseq, dseq, dc, msg)
 	case bytes.Equal(msg, AckProgress):
 		o.progressUpdate(sseq)
-	case bytes.Equal(msg, AckTerm):
```

Comment:
I didnot realize this will land in 2.10.

Concern here is that if on a < 2.10.4 the term ack is sent it will be ignored - as in the entire ack will be ignored not just the reason -  since this check here is `Equal` and term with reason wont be equal.

So I'd be curous on thoughts here on when do we surface it in clients @derekcollison @Jarema or any thoughts on how to do this while being backward compat

Reply:
This doesn't have to go into 2.10, we can just not cherry-pick it into the 2.10.x release branches.

Reply:
Unfortunately its already in 2.10.4 by the looks of things :(